app/assets/stylesheets/application.scss

```
@@ -35,3 +48,12 @@ body { padding-top: 80px; }
 .p-detail {
   margin-bottom: 0px;
 }
+.contact-detail {
+  position:absolute;
+  bottom:70px;
+}
+
+.detail{
```

Mesma coisa com relação ao espaço.

```suggestion
.detail {
```